tests/test_redispersistence.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
Im not sure I like the design of this test. You don't check at all if it works with the pip library you rely on. And therefore you dont check if the url attribute works.

The TestPostgresPersistence test is way longer, maybe it makes sense to take inspiration from there? I understand tests that you take every use case you can have with your code and write a a check for it.